modules/aws_k8s_base/tf_module/variables.tf

```diff
@@ -15,6 +15,7 @@ locals {
     ssl-redirect : true
     force-ssl-redirect : true
   }), var.nginx_config)
+  load_balancer_name = "opta-${substr(var.layer_name, 0, 22)}-lb"
```

Comment:
any reasons why we are using substrings?
And better if we need such a limit, it should go directly on the `layer_name`

Reply:
because I got errors that the lb name was too long. I can't change to a new name cause that would break everybody, so I opted out for this one, which will keep everyone's name as is (because if they're longer than 22 characters they would have failed)

Reply:
Then this is still a breaking change for existing users, as this could potentially change the Load balancer url for existing users.
Need to mention this in the Release Docs as such.